src/redux/doctors/DoctorList.js

```diff
@@ -0,0 +1,30 @@
+import { loadStorage } from '../../storage/storage';
+
+const FETCH_DOCTOR = 'HotelAgentFrontEnd/MostRecent/FETCH_DOCTOR';
+const initialState = { status: 'Not Fetched', data: [] };
+
+export const success = (mostRecent) => ({
+  type: FETCH_DOCTOR,
+  mostRecent,
+});
+
+export const fetchDoctors = () => async (dispatch) => {
+  fetch(`http://127.0.0.1:3000/v1/users/${loadStorage()}/doctors`)
+    .then((data) => data.json())
+    .then((data) => {
+      dispatch(success(data));
+    })
+    .catch((error) => {
+      throw error;
+    });
+};
+
+const doctorFetchReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_DOCTOR:
+      return { data: action.mostRecent, status: 'Date fetch Successfully!' };
+    default:
+      return state;
+  }
+};
+export default doctorFetchReducer;
```

Comment:
Kindly remove this part of your work. Maybe you did it for test purposes but this is not part of your tasks. Kindly fix that to avoid any confusion. 🙏🏾

Reply:
Hi @kosher9 thank you for time to review, I think this is my tasks to load data from doctor table to create an appointment, if we didn't have load data from doctor how can we create appointment 
it's  one of necessary partn to I did my tasks

Reply:
Hey @samiullahbahadur , I got you. So I guess the way you are doing it requires the doctors to be loaded. It is okay. But since it is related to the creation of the reservation process, you should do it in the corresponding redux folder. Because I will probably be working on that part, it could lead to some conflicts.

Reply:
Hi @kosher9  you right, but I Created you or @moise-mulungu  can replace with your tasks